packages/@okta/vuepress-site/docs/reference/error-codes/index.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: Example Error Codes and Descriptions
-excerpt: Understand Okta API errors.
+title: Okta API Error Codes
+meta:
+  - name: description
+    content: Here you can find further information about the errors that the Okta API returns, sorted by error code and HTTP return code.
 ---
 
-# Example Error Codes and Descriptions
+# Okta Error Code Overview
```

Comment:
The original wording here, specifying that these are just examples, was intentional. We needed to try to make it clear that the list of errors is not complete.

Reply:
OK - I'll change it to "Okta Example Error Codes and Descriptions"